src/view/screens/Search/Search.tsx

```diff
@@ -330,6 +363,10 @@ export function SearchScreenInner({
   const setDrawerSwipeDisabled = useSetDrawerSwipeDisabled()
   const {hasSession} = useSession()
   const {isDesktop} = useWebMediaQueries()
+  const {i18n} = useLingui()
+
+  // New Search feature gate
+  const isNewSearch = useGate('new_search')
```

Comment:
Nit: could add these gates as `const` values in a file near StatSig stuff. May be nice for features that span lots of components.

Reply:
Good idea! done